### ISSUE TITLE
Tensorboard Losses Explanination

### ISSUE THREAD

**Issue by MrAsimZahid** [tensorflow/models#9400]
Could anyone please explain these different losses including Loss/normalized_total_loss?
I am unable to found any documentation on these.
Thank you.
<img width="1056" alt="tensorLosses" src="https://user-images.githubusercontent.com/23510373/96611860-8bc3b800-1316-11eb-92e0-d613c8ede030.png">

**Comment by ravikyram** [comment]
@MrAsimZahid 

All the loss functions are defined[ here.](https://keras.io/api/losses/).Did you define your own custom loss function as normalized_total_loss. Can you please elaborate the issue. Thannks!

**Comment by MrAsimZahid** [comment]
I didn't define any custom loss function.
Thank you for sharing the link. 
Issue cleared.

**Comment by ravikyram** [comment]
@MrAsimZahid 

Please, close this thread if you feel your issue is cleared. Thanks!
